### PR TITLE
Statements make date params optional

### PR DESCRIPTION
**Pull request by AnnaGolosova**
## Link to JIRA ticket

Add a link to the JIRA ticket that the changes in this PR describe.

### *What is the problem we're trying to solve*

Add a summary of the problem and any additional context that helps explain what the issue we're trying to solve is.

### *What changes have we introduced*

Add descriptions of what changes were made to address the problem and provide a solution.

Include any links to commits that introduce significant additions of code if they help explain the process of coming to the solution. e.g Addition of test database setup, addition of first test and production class, removal of buggy code, etc.

#### _Checklist_

- [ ] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [ ] Added tests to cover all new production code
- [ ] Added comments to the README or updated relevant documentation _(add link to documentation)_, where necessary.
- [ ] Checked all code for possible refactoring
- [ ] Code pipeline builds correctly

### *Follow up actions after merging PR*

Are there any next steps that need to addressed after merging this PR? Add them here.

### CODE REVIEW

File: FinancialSummaryApi/V1/Controllers/WeeklySummaryController.cs [hunk 1]
```diff
@@ -43,7 +45,9 @@ public class WeeklySummaryController : BaseController
         [ProducesResponseType(typeof(BaseErrorResponse), StatusCodes.Status500InternalServerError)]
         [HttpGet]
         [Route("{id}")]
-        public async Task<IActionResult> Get([FromRoute] Guid id)
+        public async Task<IActionResult> Get([FromHeader(Name = "Authorization")] string token,
```
Comment (sandipchandra) [inline FinancialSummaryApi/V1/Controllers/WeeklySummaryController.cs:48]: In my view there is no need to take the token from header as the token will be validated in the API gateway by aws.

**Comment by AnnaGolosova** [comment]
Created PR to master branch by mistake

File: FinancialSummaryApi/V1/Infrastructure/Entities/AssetSummaryDbEntity.cs [hunk 2]
```diff
@@ -1,11 +1,32 @@
...
+        public SummaryType SummaryType { get; set; }
+
+        [DynamoDBProperty(AttributeName = "target_id")]
+        [DynamoDBGlobalSecondaryIndexHashKey("target_id_dx")]
+        public Guid TargetId { get; set; }
+
+        [DynamoDBProperty(AttributeName = "target_type", Converter = typeof(DynamoDbEnumConverter<TargetType>))]
+        public TargetType TargetType { get; set; }
+
+        [DynamoDBProperty(AttributeName = "submit_date", Converter = typeof(DynamoDbDateTimeConverter))]
```
Comment (sandipchandra) [inline FinancialSummaryApi/V1/Infrastructure/Entities/AssetSummaryDbEntity.cs:24]: May be we need to use the Hackney Nuget Packages as currently they are forcing to use them.

**Review by sandipchandra — COMMENTED** [review]
Put some suggestions.

**Pull request closed without merging**